lib/pages/chat/events/html_message.dart

```diff
@@ -163,6 +169,20 @@ class HtmlMessage extends StatelessWidget {
         }
         return {};
       },
-    );
+      pillBuilder: (identifier, url, onTap, getMxcUrl) {
+        debugPrint('identifier: $identifier, url: $url');
+        if (event!.status == EventStatus.sending) {
+          chatController!.storePillSendingEventId(event!.eventId);
+        }
+        if (event!.status == EventStatus.sending || chatController!.isSyncEventForSendingPill(event: event!)) {
```

Comment:
case A: ` if (event!.status == EventStatus.sending)`
case B: ` if (event!.status == EventStatus.sending || chatController!.isSyncEventForSendingPill(event: event!))`

If (event!.status == EventStatus.sending) Will this case happen first? Not clear to me, plss specifically explain to me

Reply:
When the event's status is 'Sending,' it indicates that the message is currently being sent. I need to store its eventId, and it will later appear in the 'synced' event once the message is successfully sent. This is why I have 'Case A.' 'Case B' refers to two scenarios: when the event is in the 'Sending' status or when its status is 'synced' (synced specifically for this message).

Reply:
deserve a method for it :)